tests/conftest.py

```diff
@@ -1,122 +1,55 @@
+"""Conftest module."""
+from asyncio import AbstractEventLoop
+import os
```

Comment:
Har du bruk for denne da linje under også er import fra samme pakke

Reply:
bruker `os.path` på linje 46, men er vel lite vits med egen import for env. Fjerner

Reply:
ea551a8